src/Sylius/Bundle/ProductBundle/Resources/config/serializer/Model.ProductOptionValue.yml

```diff
@@ -5,9 +5,8 @@ Sylius\Component\Product\Model\ProductOptionValue:
         code:
             expose: true
             type: string
-        value:
-            expose: true
-            type: string
     virtual_properties:
         getName:
             serialized_name: name
+        getValue:
```

Comment:
Same as above. Both virtual properties should be removed and we should expose all translations